core/src/test/scala/org/apache/spark/sql/store/CreateIndexTest.scala

```diff
@@ -69,6 +73,7 @@ class CreateIndexTest extends SnappyFunSuite with BeforeAndAfterEach {
   test("Test choice of index according to where predicate") {
     val tableName = "tabOne"
     val snContext = SnappyContext(sc)
+    snContext.setConf(io.snappydata.Property.EnableExperimentalFeatures.configEntry.key, "true")
```

Comment:
.name instead of .configEntry.key will be good. Lets use that so it becomes a general practise for future code.